nextflow_schema.json

```
@@ -764,6 +764,17 @@
                     "description": "Maximal number of contigs that are not part of any bin but treated as individual genome.",
                     "help_text": "Contigs that do not fulfill the thresholds of `--min_length_unbinned_contigs` and `--max_unbinned_contigs` are pooled for downstream analysis and reporting, except contigs that also do not fullfill `--min_contig_size` are not considered further."
                 },
+                "bin_min_size": {
+                    "type": "integer",
+                    "default": 0,
+                    "description": "Specify the shortest length a bin should be to retain for downstream processing (in base pairs)",
+                    "help_text": "This can speed up downstream processing by removing 'nonsense' bins (e.g. badly binned single short-contig bins, or unnaturally large bins). "
+                },
+                "bin_max_size": {
+                    "type": "integer",
```

And again:

```suggestion
                    "type": "integer",
                    "minimum": 0,
```

Maybe the minimum could be higher? I don't know if it's possible to set in the schema but we could also enforce that if not null, it has to be bigger than `bin_min_size`?